main/acle.md

```diff
@@ -300,9 +416,9 @@ defining how they are dealt with by the AAPCS.
 
 >   - Vector types for use with the Advanced SIMD intrinsics (see
 >     [ssec-vectypes](#vector-data-types)).
->   - The <span class="title-ref">\_\_fp16</span> type for 16-bit
+>   - The ``\_\_fp16`` type for 16-bit
```

Comment:
Nice, but as you can see in [this rendering](https://github.com/ValeriaDaneva/acle/blob/github.io-convert-acle-rst-to-md/main/acle.md#data-types), we need to get rid of all the extra slash symbols. This should render as `__fp16`, not as `\_\_fp16`. This is true also for `\<` and `\>` I have seen used in other places.

Also, to render typewrite font, all you need is to wrap it in single `, not double